README.md

```diff
@@ -72,10 +46,12 @@ Additionally, the extension will monitor **Package.swift** and **Package.resolve
 
 > **Note**: When browsing the files in a package, Visual Studio Code may also open these files in the Explorer. If this is undesirable, open **Preferences ▸ Settings** and set **Explorer: Auto Reveal** to `false`.
 
-## Configuration
```

Comment:
We should at least point users in the right direction then, because the Marketplace doesn't show the Feature Contributions tab.

Reply:
I'm not sure users need to see configuration options when viewing in the marketplace.